src/raftstore/store/peer.rs

```diff
@@ -608,14 +608,14 @@ impl Peer {
     }
 
     #[inline]
-    fn send<T, I>(&mut self, trans: &T, msgs: I, metrics: &mut RaftMessageMetrics) -> Result<()>
+    fn send<T, I>(&mut self, trans: &mut T, msgs: I, metrics: &mut RaftMessageMetrics)
```

Comment:
Why mut?

Reply:
Otherwise can't pass compiling.

Reply:
Why?

Reply:
`send_raft_msg` in master needs a mut `T`. However I found we can remove the `mut`.